packages/elements/src/interactive-chart/index.ts

```diff
@@ -790,28 +767,11 @@ export class InteractiveChart extends ResponsiveElement {
          * Don't need to be updated if chart has no data.
          */
         /* c8 ignore start */
-        let value;
+        let value: SeriesDataItem | string | number | undefined;
```

Comment:
no need for `number` here, right?
```suggestion
        let value: SeriesDataItem | string | undefined;
```